src/chains-config/index.ts

```diff
@@ -29,6 +31,8 @@ const specToControllerMap = {
 	'dock-main-runtime': dockMainnetControllers,
 	statemint: statemintControllers,
 	statemine: statemineControllers,
+	westmine: westmineControllers,
```

Comment:
Might DRY things up a bit if we had all these keys point to the same values - just means if those parachains ever diverged we would need to re-evaluate controller configs; but we will need to do that anyways.

Reply:
Ahhh, I see what you are saying. Good point. Yea Ill fix that up and make them one value. Also will definitely lower the amount of files we have which are preferred.